globals.go

```diff
@@ -522,6 +528,14 @@ var (
 
 // permission levels
 var (
+	AdminSettingsEdit = PermissionLevel{
+		Description: "Edit admin settings",
+		Value:       10,
+	}
+	ProjectSettingsCreate = PermissionLevel{
```

Comment:
This should be a separate permission, rather than a new level on the existing permission